src_python/habitat_sim/utils/datasets_download.py

```diff
@@ -109,16 +109,16 @@ def initialize_test_data_sources(data_path):
             "version": "1.0",
         },
         "replica_cad_dataset": {
-            "source": "https://dl.fbaipublicfiles.com/habitat/ReplicaCAD/ReplicaCAD_dataset_v1.3.zip",
-            "package_name": "ReplicaCAD_dataset_v1.3.zip",
+            "source": "https://dl.fbaipublicfiles.com/habitat/ReplicaCAD/ReplicaCAD_dataset_v1.4.zip",
+            "package_name": "ReplicaCAD_dataset_v1.4.zip",
             "link": data_path + "replica_cad",
-            "version": "1.3",
+            "version": "1.4",
         },
         "replica_cad_baked_lighting": {
-            "source": "https://dl.fbaipublicfiles.com/habitat/ReplicaCAD/ReplicaCAD_baked_lighting_v1.3.zip",
-            "package_name": "ReplicaCAD_baked_lighting_v1.3.zip",
+            "source": "https://dl.fbaipublicfiles.com/habitat/ReplicaCAD/ReplicaCAD_baked_lighting_v1.4.zip",
+            "package_name": "ReplicaCAD_baked_lighting_v1.4.zip",
             "link": data_path + "replica_cad_baked_lighting",
-            "version": "1.3",
+            "version": "1.4",
```

Comment:
Let's keep these changes isolated to #1694, so remove before merge.

Reply:
Done